common/test/acceptance/tests/studio/test_studio_settings_certificates.py

```diff
@@ -52,7 +52,7 @@ def create_and_verify_certificate(self, course_title_override, existing_certs, s
             certificate.signatories[idx].name = signatory['name']
             certificate.signatories[idx].title = signatory['title']
             certificate.signatories[idx].organization = signatory['organization']
-            certificate.signatories[idx].upload_signature_image('Signature-{}.png'.format(idx))
+            certificate.signatories[idx].upload_signature_image('Signature_{}.png'.format(idx))
```

Comment:
I think you should changed the size of existing `Signature-0.png` and `Signature-1.png` instead of creating new images.